cogeo_mosaic/backends/dynamodb.py

```diff
@@ -31,14 +34,38 @@ class DynamoDBBackend(BaseBackend):
     client: Any = attr.ib(default=None)
     region: str = attr.ib(default=os.getenv("AWS_REGION", "us-east-1"))
     table_name: str = attr.ib(init=False)
+    mosaic_name: str = attr.ib(init=False)
     table: Any = attr.ib(init=False)
 
     _backend_name = "AWS DynamoDB"
+    _metadata_quadkey: str = "-1"
 
     def __attrs_post_init__(self):
-        """Post Init: parse path, create client and connect to Table."""
+        """Post Init: parse path, create client and connect to Table.
+
+        A path looks like
+
+        dynamodb://{region}/{table_name}:{mosaic_name}
+        dynamodb:///{table_name}:{mosaic_name}
+
+        """
+        logger.debug(f"Using DynamoDB backend: {self.path}")
+
+        if not re.match(
+            r"dynamodb://([a-z]{2}\-[a-z]+\-[0-9])?\/[a-zA-Z0-9\_\-\.]+\:[a-zA-Z0-9\_\-\.]+$",
```

Comment:
So a non-empty `mosaic_name` is required right?

Reply:
yes